src/components/reimbursement_card.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import Image from "next/image";
+
+// Temporary Reimbursement type
+// Will later use Reimbursement schema
+export type ReimbursementInfo = {
```

Comment:
The Reimbursement schema should be merged into develop so if you pull those recent changes you should be able to use the schema now